chromium_src/chrome/browser/safe_browsing/safe_browsing_service.cc

```diff
@@ -12,7 +12,8 @@ namespace safe_browsing {
 
 const scoped_refptr<SafeBrowsingUIManager>&
 SafeBrowsingService::ui_manager() const {
-  return nullptr;
+  static scoped_refptr<SafeBrowsingUIManager> empty;
```

Comment:
we have several of these warnings and I don't think this is necessary right now

Reply:
There is only one left besides this to get a warning-free compilation, would you prefer to leave it as is? I guess there is no harm in fixing them.